tests/test_configs/odqa/en_odqa_infer_wiki_test.json

```diff
@@ -48,7 +48,7 @@
       {
         "class_name": "logit_ranker",
         "batch_size": 10,
-        "squad_model": {"config_path": "{CONFIGS_PATH}/squad/multi_squad_noans_infer.json"},
+        "squad_model": {"config_path": "{CONFIGS_PATH}/squad/multi_squad_bert_noans.json"},
```

Comment:
remove all three odqa test configs and replace them with non-test versions in the test_quick_start